res/database/Default/database_news.xml

```diff
@@ -6904,22 +6982,45 @@ W tym roku ponownie spodziewane są demonstracje.</pl>
 			<data genre="1" price="1.0" quality="35" />
 			<availability year_range_from="-1" year_range_to="-1" />
 		</news>
-		<news guid="182fc068-6f07-45be-bc60-5637ce50f6e4" thread_guid="ffb5a2fd-d41c-4f8b-a3ec-546762485d47" type="0" creator="">
+		<news guid="182fc068-6f07-45be-bc60-5637ce50f6e4" thread_guid="ffb5a2fd-d41c-4f8b-a3ec-546762485d47" type="0" creator="" comment="TODO: Could add other sports">
 			<title>
-				<de>${borisbecker} plant Comeback</de>
-				<en>${borisbecker} plans comeback</en>
-				<pl>${borisbecker} planuje powrót</pl>
+				<de>${athletefirstname} ${athletelastname} plant Comeback</de>
+				<en>${athletefirstname} ${athletelastname} plans comeback</en>
+				<pl>${athletefirstname} ${athletelastname} planuje powrót</pl>
 			</title>
 			<description>
-				<de>Nach jahrelanger Tennis-Abstinenz plant ${borisbecker} sein Comeback. Konkurrenten freuen sich auf legendäre Hechtsprünge.</de>
-				<en>After years of tennis abstinence, ${borisbecker} is planning his comeback. Competitors are looking forward to legendary pike jumps.</en>
-				<pl>Po latach tenisowej abstynencji ${borisbecker} planuje swój powrót. Zawodnicy nie mogą się doczekać legendarnych skoków szczupakiem.</pl>
+				<de>Nach jahrelanger Tennis-Abstinenz plant ${athletefirstname} ${athletelastname} sein Comeback. Konkurrenten freuen sich ${legendary_${epoch}}.</de>
+				<en>After years of tennis abstinence, ${athletefirstname} ${athletelastname} is planning his comeback. Competitors are looking forward ${legendary_${epoch}}.</en>
+				<pl>Po latach tenisowej abstynencji ${athletefirstname} ${athletelastname} planuje swój powrót. Zawodnicy nie mogą się doczekać ${legendary_${epoch}}.</pl>
 			</description>
 			<variables>
-				<borisbecker>${.person:"209467cd-85e1-4212-999e-b6273359320f":"fullname"}</borisbecker>
+				<athletesfirstname_csv>${.person:"614c90b8-e18c-45d2-a026-a3d216b276cb":"firstname"};${.person:"c6f7756d-3e0c-4e47-9cbe-13dc7826e145":"firstname"};${.person:"b073ffa5-ad2f-40ce-ba52-b5b9d5e4a2a6":"firstname"};${.person:"209467cd-85e1-4212-999e-b6273359320f":"firstname"}</athletesfirstname_csv>
```

Comment:
You could just put the GUID in the csv and use a single expression for first- and last name. This would increase readability and possibly consistency as both GUID and function will not be duplicated.

Reply:
i do this sometimes, i'm still experimenting with different approaches in such cases.